src/user/controller/by-href.ts

```diff
@@ -13,25 +13,24 @@ class UserByHrefController extends Controller {
 
     let hasControl = false;
     let hasPassword = false;
-    let hasPrivilege = false;
+    const isAdmin = await privilegeService.hasPrivilege(ctx, 'admin');
+
     if (ctx.state.user.id === user.id) {
       hasControl = true;
     } else if (await privilegeService.hasPrivilege(ctx, 'admin')) {
```

Comment:
This line should use the new flag instead of doing the same operation again, so we do fewer database queries.